src/chocolatey/infrastructure/filesystem/IFileSystem.cs

```diff
@@ -351,6 +381,15 @@ public interface IFileSystem
         /// <param name="overwriteExisting">true if the destination directory can be overwritten; otherwise, false.</param>
         void CopyDirectory(string sourceDirectoryPath, string destinationDirectoryPath, bool overwriteExisting);
 
+        /// <summary>
+        ///   Copies an existing directory to a new directory. Overwriting a directory of the same name is allowed.

```

Comment:
Do we need to extend this regarding the ability to make it silent?

Reply:
I don't think this is needed, as it is mentioned as the documentation in the parameters.